### PR TITLE
Hw4

### PR DESCRIPTION
**Pull request by lyalchenko1995**
Hw4

### CODE REVIEW

File: src/test/java/com/epam/tc/hw2/ex1/TestsExercise1.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public void testLogin() {
         findElementBySelector(By.id("login-button")).click();
 
         WebElement user = findElementBySelector(By.id("user-name"));
-        new WebDriverWait(webDriver, Duration.ofSeconds(5)).until(ExpectedConditions.visibilityOf(user));
+        //new WebDriverWait(webDriver, Duration.ofSeconds(5)).until(ExpectedConditions.visibilityOf(user));
```
Comment (komyagin) [inline src/test/java/com/epam/tc/hw2/ex1/TestsExercise1.java]: Please remove this comment line

File: src/test/java/com/epam/tc/hw2/ex2/TestsExercise2.java [hunk 2]
```diff
@@ -41,7 +41,7 @@ public void testLogin() {
         findElementBySelector(By.id("login-button")).click();
 
         WebElement user = findElementBySelector(By.id("user-name"));
-        new WebDriverWait(webDriver, Duration.ofSeconds(5)).until(ExpectedConditions.visibilityOf(user));
+        //new WebDriverWait(webDriver, Duration.ofSeconds(5)).until(ExpectedConditions.visibilityOf(user));
```
Comment (komyagin) [inline src/test/java/com/epam/tc/hw2/ex2/TestsExercise2.java]: Please remove this comment line

**Review by komyagin — CHANGES_REQUESTED** [review]

File: src/test/java/com/epam/tc/hw2/ex1/TestsExercise1.java (hunk 1 above)
Comment (lyalchenko1995) [reply src/test/java/com/epam/tc/hw2/ex1/TestsExercise1.java]: done

File: src/test/java/com/epam/tc/hw2/ex2/TestsExercise2.java (hunk 2 above)
Comment (lyalchenko1995) [reply src/test/java/com/epam/tc/hw2/ex2/TestsExercise2.java]: done

**Review by komyagin — APPROVED** [review]

**Pull request merged**